package com.ocp.exceptions6;

import java.io.FileNotFoundException;
import java.io.IOException;

public class ExceptionTest18 {
	public static void main(String[] seatbelts) throws Exception {  // w1
	     /* try {
	         throw new IOException("Auto-pilot error");
	      } catch (Exception | CarCrash e) {  // w2
	         throw e;
	      } catch (Exception a) {  // w3
	         throw a;
	      }*/
	   }
}

// Ans:The code does not compile because of line w2.




